WikipediaUnitTests/Code/MWKImageInfo+MWKImageComparisonTests.m

```diff
@@ -21,16 +13,16 @@ @interface MWKImageInfo_MWKImageComparisonTests : XCTestCase
 @implementation MWKImageInfo_MWKImageComparisonTests
 
 - (void)testAssociation {
-    MWKImage* image    = [MWKImage imageAssociatedWithSourceURL:@"some_file_name.jpg/400px-some_file_name.jpg"];
-    MWKImageInfo* info = [MWKImageInfo infoAssociatedWithSourceURL:@"some_file_name.jpg/800px-some_file_name.jpg"];
+    MWKImage* image    = [MWKImage imageAssociatedWithSourceURL:@"/thumb/some_file_name.jpg/400px-some_file_name.jpg"];
```

Comment:
@coreyfloyd the tests were failing. They're failing because they aren't using actual URLs, but rather a made up partial path. In the wild, URLs like this would have /thumb/ somewhere in the path. If we want to go a step further, we can use actual commons thumbnail URLs to test.